papermill/tests/test_cli.py

```diff
@@ -357,6 +373,23 @@ def test_no_report_mode(self, execute_patch):
             progress_bar=True,
             start_timeout=60,
             report_mode=False,
+            version=1.0,
+        )
+
+    @patch(cli.__name__ + '.execute_notebook')
+    def test_version(self, execute_patch):
+        self.runner.invoke(papermill, self.default_args + ['--version'])
+        execute_patch.assert_called_with(
```

Comment:
If we do this right, `execute_notebook` should not be called with `--version`. Test here should say `execute_patch.assert_not_called()` when it's working.